torch_xla/distributed/fsdp/xla_fully_sharded_data_parallel.py

```diff
@@ -398,6 +406,20 @@ def __init__(
     # When `_shard_param_on_dim_0` is True, we shard and all-gather model parameter tensors
     # only along their dim 0 without flattening the parameter
     self._shard_param_on_dim_0 = shard_param_on_dim_0 and not flatten_parameters
+    # Allow specifying groups for the sharding collective ops, useful for mixing
```

Comment:
this prs changes default behavior of the FSDP, if you want to merge this part, please make sure proper tests are added. @alanwaketan can you review the FSDP part of codes?

Reply:
Will post a separate PR for the FSDP change.